latexify.py

```diff
@@ -50,10 +52,10 @@ def visit_Num(self, node):
 
   def visit_UnaryOp(self, node):
     def _wrap(child):
-      repr = self.visit(child)
+      repr_ = self.visit(child)
```

Comment:
I guess just `latex` is better than `repr`.

Reply:
Done.